core-foundation/src/string.rs

```diff
@@ -136,10 +137,8 @@ impl CFString {
 
     /// Returns the number of characters in the string.
     #[inline]
-    pub fn char_len(&self) -> CFIndex {
-        unsafe {
-            CFStringGetLength(self.0)
-        }
+    pub unsafe fn char_len(&self) -> CFIndex {
+        CFStringGetLength(self.0)
```

Comment:
Why is this function made unsafe as a whole? Is `CFStringGetLength` not a safe interface?